src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -561,6 +561,10 @@ private int getLimit(boolean includeItem) {
       case 404:
         result = Future.succeededFuture(PostPatronAccountInstanceHoldByIdAndInstanceIdResponse.respond404WithTextPlain(message));
         break;
+        case 422:
+          result = Future.succeededFuture(PostPatronAccountInstanceHoldByIdAndInstanceIdResponse.
+            respond422WithApplicationJson(new Errors().withErrors(List.of(new Error().withMessage(message)))));
+          break;
```

Comment:
The indentation of this case seems to be different to the one above it. Should it be consistent?

Reply:
Yes I will fix it as soon will have access to the repo.